CHANGELOG.md

```diff
@@ -1,3 +1,76 @@
+# Release 0.19.1
+
+# What's New
+
+* Ziti CLI now has 'Let's Encrypt' PKI support to facilitate TLS connections to Controller from
+  BrowZer-based apps that use the `ziti-sdk-js`.
+
+    * New command to Register a Let's Encrypt account, then create and install a certificate
+
+        Usage:
+
+        `ziti pki le create -d domain -p path-to-where-data-is-saved [flags]`
+
+        Flags:
+
+          -a, --acmeserver string                             ACME CA hostname (default "https://acme-v02.api.letsencrypt.org/directory")
+          -d, --domain string                                 Domain for which Cert is being generated (e.g. me.example.com)
+          -e, --email string                                  Email used for registration and recovery contact (default "openziti@netfoundry.io")
+          -h, --help                                          help for create
+          -k, --keytype EC256|EC384|RSA2048|RSA4096|RSA8192   Key type to use for private keys (default RSA4096)
+          -p, --path string                                   Directory to use for storing the data
+          -o, --port string                                   Port to listen on for HTTP based ACME challenges (default "80")
+          -s, --staging                                       Enable creation of 'staging' Certs (instead of production Certs)
```

Comment:
Is there any ability to add additional SANs?

Reply:
Not at the moment. What did you have in mind?

Reply:
SANs help with moving between domains, ip, etc. And some other network tools use SANs as a way to pass identity information. More of a nice to have if we can type thing.

Reply:
Please open an issue to track that ask/need, and we can deal with it separate from this initial PR.